<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Module</class>
 <widget class="QWidget" name="Module">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>862</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1042</width>
        <height>850</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="38" column="3">
        <widget class="QLabel" name="mesa3D">
         <property name="text">
          <string>Mesa 3D:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="46" column="3">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Build Version:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="32" column="3">
        <widget class="QLabel" name="memory">
         <property name="text">
          <string>Memory:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="46" column="4">
        <widget class="QLabel" name="buildLabel">
         <property name="text">
          <string>{BuildVersion}</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="12" column="4">
        <widget class="QLabel" name="frameworksLabel">
         <property name="text">
          <string>{FrameworksVersion}</string>
         </property>
        </widget>
       </item>
       <item row="49" column="4">
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="10" column="3">
        <widget class="QLabel" name="plasma">
         <property name="text">
          <string>KDE Plasma Version:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="13" column="4">
        <widget class="QLabel" name="qtLabel">
         <property name="text">
          <string notr="true">{QtVersion}</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="13" column="3">
        <widget class="QLabel" name="qt">
         <property name="text">
          <string>Qt Version:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="39" column="3">
        <widget class="QLabel" name="displayServerVersion">
         <property name="text">
          <string>Display Server:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="49" column="5">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="14" column="4">
        <widget class="QLabel" name="kernelLabel">
         <property name="text">
          <string notr="true">{KernelName}</string>
         </property>
        </widget>
       </item>
       <item row="44" column="3">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>OS Type:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="42" column="4">
        <widget class="QLabel" name="networkLabel">
         <property name="text">
          <string>{OnlineStatus}</string>
         </property>
        </widget>
       </item>
       <item row="11" column="4">
        <widget class="QLabel" name="appsLabel">
         <property name="text">
          <string>{AppsVersion}</string>
         </property>
        </widget>
       </item>
       <item row="11" column="3">
        <widget class="QLabel" name="apps">
         <property name="text">
          <string>KDE Apps Version:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="49" column="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="12" column="3">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Frameworks Version:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="43" column="4">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>OS</string>
         </property>
        </widget>
       </item>
       <item row="10" column="4">
        <widget class="QLabel" name="plasmaLabel">
         <property name="text">
          <string notr="true">{KDEVersion}</string>
         </property>
        </widget>
       </item>
       <item row="36" column="4">
        <widget class="QLabel" name="openglDriverLabel">
         <property name="text">
          <string>{OpenGL Driver}</string>
         </property>
        </widget>
       </item>
       <item row="3" column="8">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="7">
        <widget class="QLabel" name="softwareLogo">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="36" column="3">
        <widget class="QLabel" name="openglDriver">
         <property name="text">
          <string>OpenGL Driver:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Software</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="30" column="4">
        <widget class="QLabel" name="processorLabel">
         <property name="text">
          <string notr="true">{ProcessorName}</string>
         </property>
        </widget>
       </item>
       <item row="39" column="4">
        <widget class="QLabel" name="displayServerVersionLabel">
         <property name="text">
          <string>{Display Server Version}</string>
         </property>
        </widget>
       </item>
       <item row="34" column="4">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Graphics</string>
         </property>
        </widget>
       </item>
       <item row="30" column="3">
        <widget class="QLabel" name="processor">
         <property name="text">
          <string notr="true">Processor:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="41" column="4">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Network</string>
         </property>
        </widget>
       </item>
       <item row="49" column="7">
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="41" column="7">
        <widget class="QLabel" name="networkLogo">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="21" column="7">
        <widget class="QLabel" name="hardwareLogo">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="34" column="7">
        <widget class="QLabel" name="graphicsLogo">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="4" colspan="2">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <spacer name="verticalSpacer_6">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="logoLabel">
              <property name="text">
               <string notr="true">{LogoLabel}</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="nameVersionLabel">
              <property name="text">
               <string notr="true">{NameVersionLabel}</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="urlLabel">
              <property name="text">
               <string notr="true">{UrlLabel}</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="8">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="14" column="3">
        <widget class="QLabel" name="kernel">
         <property name="text">
          <string>Kernel Version:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="32" column="4">
        <widget class="QLabel" name="memoryLabel">
         <property name="text">
          <string notr="true">{MemoryAmount}</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="3">
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="35" column="3">
        <widget class="QLabel" name="opengl">
         <property name="text">
          <string>OpenGL Renderer:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="35" column="4">
        <widget class="QLabel" name="openglLabel">
         <property name="text">
          <string>{OpenGLRenderer}</string>
         </property>
        </widget>
       </item>
       <item row="44" column="4">
        <widget class="QLabel" name="bitsLabel">
         <property name="text">
          <string notr="true">{QtArchitecutre}</string>
         </property>
        </widget>
       </item>
       <item row="21" column="4">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Hardware</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="42" column="3">
        <widget class="QLabel" name="network">
         <property name="text">
          <string>Network Status:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="45" column="3">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>OS Version:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="45" column="4">
        <widget class="QLabel" name="namedReleaseLabel">
         <property name="text">
          <string>{OSVersion}</string>
         </property>
        </widget>
       </item>
       <item row="38" column="4">
        <widget class="QLabel" name="mesaVersionLabel">
         <property name="text">
          <string>{Mesa Version}</string>
         </property>
        </widget>
       </item>
       <item row="33" column="3">
        <widget class="QLabel" name="swapLabel">
         <property name="text">
          <string>Swap:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="33" column="4">
        <widget class="QLabel" name="swapAmount">
         <property name="text">
          <string>{SwapAmount}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
